websites/J/Jet Set Radio Live/dist/metadata.json

```
@@ -16,7 +16,7 @@
     "nl": "Yo! Je hebt toch van Jet Set Radio gehoord? Het is alleen het populairste radiostation dat rechtstreeks uit Tokio komt! Stem af. Zet aan. Jetset Radiooo !!!"
   },
   "url": "jetsetradio.live",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "logo": "https://i.imgur.com/ZiIN6t9.png",
   "thumbnail": "https://i.imgur.com/TmgNbn4.jpeg",
```

`.jpeg` isn't a valid filetype, must be `.png` or `.jpg`, hence why it failed schema validation.

Use this
```suggestion
  "thumbnail": "https://i.imgur.com/TmgNbn4.png",
```

---

Technically this should be a valid filetype in the schema since it is an actual valid extension, I've updated this regardless.